circuits/cpp/barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/eccvm/eccvm_circuit_builder.test.cpp

```diff
@@ -0,0 +1,273 @@
+#include "barretenberg/crypto/generators/generator_data.hpp"
```

Comment:
There is no check for reset opcode

Reply:
The `eq` circuit builder method actually creates both an `eq` and a `reset` opcode. I've renamed to `eq` to `eq_and_reset` to better describe its function.